src/Fmt.ml

```diff
@@ -127,8 +127,12 @@ let wrap_k x = wrap_if_k true x
 
 let wrap_if cnd pre suf = wrap_if_k cnd (fmt pre) (fmt suf)
 
+and wrap_if_str cnd pre suf = wrap_if_k cnd (str pre) (str suf)
+
```

Comment:
No longer needed externally, and only one use below: better to remove and inline.